core/src/subgraph/instance_manager.rs

```diff
@@ -746,6 +750,7 @@ where
 
         // Transact entity operations into the store and update the
         // subgraph's block stream pointer
+        let _section = ctx.host_metrics.stopwatch.start_section("transact_block");
```

Comment:
Having the stopwatch stop implicitly when the section is dropped is almost too magical for my taste. I wonder if there is a better name than `start_section` that makes this more obvious. Like `measure_until_dropped` (not a great name) or something.

Reply:
The idea is to keep metrics from cluttering the code, so I figured one line of code is better than two. I'm up for suggestions on naming.

Reply:
I don't have any good ideas, so let's keep it.